contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -105,6 +110,23 @@ contract WithdrawManager is WithdrawManagerStorage, IWithdrawManager {
     _;
   }
 
+  function startExitWithDepositedTokens(uint256 depositId, address token, uint256 amountOrToken)
+    external
+    payable
+    isBondProvided
```

Comment:
why `isBondProvided `?
we need to make sure that there is at least 1 checkpoint before `ExitWithDepositedTokens`.

Reply:
This needs to be a bonded exit, since I could have spent the tokens on the side chain and start an exit here; then that needs to be challenged.

Reply:
Waiting on 1 checkpoint still sounds better than, bonded exit.
let's say user accidentally deposited wrong amount(more then needed) and he just wants the normal exit to get this amount out on top of that now also need to transfer bond amount, which will be locked for some time(in case of `ExitWithDepositedTokens` we should think of faster exit then normal plasma exit if possible).

Reply:
@jdkanani what do you think?.